service/storage_proxy.cc

```diff
@@ -721,8 +742,54 @@ class storage_proxy::remote {
         }
         shared_ptr<storage_proxy> p = _sp.shared_from_this();
         auto cmd = make_lw_shared<query::read_command>(std::move(cmd1));
+
         auto src_ip = src_addr.addr;
         auto timeout = t ? *t : db::no_timeout;
+
+        auto f_r = co_await coroutine::as_future([&]() -> future<bool> {
+            // Check if we have a forward query
+            if (!cmd->slice.is_reversed())
+                return ::make_ready_future<bool>(false);
+
+            // We deal with a reversed query. Check if read command is coming from an old node.
+            // Table schema is then passed with the read command.
```

Comment:
This sentence is ambiguous. Need to rephrase to be connected with the previous sentence:
"If coordinator is an old node, it will send the tabls schema in the read command".